index.html

```diff
@@ -45,6 +45,342 @@ <h2 class="supporting">
+                                    <li class="html">
+                                        <span>html</span>
+                                    </li>
+                                    <li class="bootstrap">
+                                        <span>bootstrap</span>
+                                    </li>
+                                    <li class="ruby">
+                                        <span>Ruby</span>
+                                    </li>
+                                </ul>
+                            </div>
+
+                        </div>
+                        <div class="action">
+                            <a href="#">
+                                <button>
+                                    <span>See Project</span>
+                                </button>
+                            </a>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </section>
+        <!-- end of works section -->
+
+        <!-- start of about me section -->
+
+        <section class="about-me-section">
+            <div class="about">
+                <span class="title">
+                    About me
+                </span>
+                <p class="description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through
+                    some of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact
+                    me.
+                </p>
+                <a href="#">
+                    <button>
+                        <span>
+                            Get My Resume
+                        </span>
+                    </button>
+                </a>
+            </div>
+            <hr class="about-skill-hr">
+            <div class="skill">
+                <div class="frame-35">
+                    <div class="frame-30">
+                        <img src="./imgs/Rectangle55.svg" alt="rectangle">
+                        <h3>Language</h3>
+                    </div>
+                    <ul class="frame-29">
+                        <li class="javascript">
+                            <h3>Javascript</h3>
+                        </li>
+                        <li class="ruby">
+                            <h3>Ruby</h3>
+                        </li>
+
+                        <li class="html">
+                            <h3>Html</h3>
+                        </li>
+                        <li class="css">
+                            <h3>Css</h3>
+                        </li>
+                    </ul>
+                </div>
+                <div class="frame-34">
+                    <div class="frame-31">
+                        <img src="./imgs/Rectangle56.svg" alt="rectangle">
+                        <h3>Frameworks</h3>
+                    </div>
```

Comment:
- [ ] Kindly make sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` for better semantics 💯 (The technology in each category (e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`)

Reply:
Done.